apps/sms/views/conversation/js/information.js

```diff
@@ -247,14 +248,17 @@ var VIEWS = {
 
         ConversationView.promptContact({
           number: target.dataset.number
-        });
+        }).then(
+          () => Navigation.toPanel('thread', { id: this.id })
+        );
```

Comment:
Ayman made it very clear that he wanted the Group View to look like a view and not a dialog :)